test/library/packages/UnitTest/PopulateTests_PRIM/TestCountInt.chpl

```diff
@@ -0,0 +1,10 @@
+proc dummy_test_int(test: int) throws {
```

Comment:
Definitely add a comment on what this file is checking.  I like the idea behind testing its behavior, but maybe we should instead generate a warning when something other than the Test class or a subclass of it is used with this primitive (since the primitive literally mentions tests)?

Reply:
I think it will be little difficult to do that for now. But later we can deal with it. If you want I will create an issue for it.

Reply:
That would be good :)

Reply:
Once this PR gets accepted, I will create an issue.